style.css

```diff
@@ -109,3 +118,231 @@ ul {
 .social-icons li {
   padding-right: 18px;
 }
+
+/* Project Section */
+.project-section {
+  display: grid;
+  padding: 12vh 8vw;
+  row-gap: 12vh;
+}
+
+.project-card {
+  background-color: white;
+  border-radius: 1em;
+  border: 1px solid #dfe1e6;
+  padding: 1em;
+}
+
+.project-img img {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+.project-card h2 {
+  font-weight: 700;
+  font-size: 2em;
+  line-height: 1.5em;
+  margin: 0.5em 0 1em;
+}
+
+.project-info {
+  display: flex;
+  align-items: baseline;
+}
+
+.project-card h3 {
+  font-weight: 600;
+  font-size: 0.9em;
+  line-height: 16px;
+  color: #344563;
+  margin: 0;
+}
+
+.project-info p {
+  font-weight: 600;
+  font-size: 0.9em;
+  line-height: 16px;
+  color: #7a869a;
+  margin: 0 0 0 10px;
+}
+
+.project-info p::before {
+  content: url("assets/Counter.svg");
+  display: inline-block;
+  margin-right: 10px;
+}
+
+.project-description {
+  font-weight: 400;
+  font-size: 1em;
+  line-height: 24px;
+  color: #344563;
+  margin: 20px 0;
+}
+
+.project-tags {
+  display: flex;
+  margin: 16px 0;
+}
+
+.project-tags a {
+  padding: 4px 12px;
+  background-color: #ebebff;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 0.8em;
+  line-height: 16px;
+  color: var(--blue);
+  margin-right: 8px;
+}
+
+.project-tags a:hover {
+  background-color: #d6d6ff;
+}
+
+.project-btn {
+  padding: 14px;
+  border: 1px solid var(--button-color);
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 1.1em;
+  line-height: 24px;
+  color: var(--button-color);
+  margin: 14px 0 18px;
+}
+
+.project-btn:active {
+  background-color: #2230d2;
+  color: white;
+  border-color: #2230d2;
+}
+
+/* ABOUT SECTION */
+.about-section {
+  margin-top: 0;
+}
+
+.about-container {
+  background-color: #fff;
+  padding: 114px 0;
+  border-radius: 0 15% 0 0;
+}
```

Comment:
Your design
![image](https://github.com/CVILLA09/Portfolio/assets/78251927/81fb72f7-0edb-40f1-8f8a-b6d5e8c1d74e)

Figma
![image](https://github.com/CVILLA09/Portfolio/assets/78251927/7f85bc4e-9455-4f57-8d9a-47e6a7fb2a7e)

I would suggest that we use pixels to enable a smooth border-radius similar to the design in Figma. 🤓

example
```CSS
border-radius: 0 80px 0 0;
```